evalml/model_understanding/prediction_explanations/explainers.py

```diff
@@ -44,13 +44,13 @@ def explain_prediction(pipeline, input_features, y, index_to_explain, top_k_feat
         ValueError: if an output_format outside of "text", "dict" or "dataframe is provided.
     """
     input_features = infer_feature_types(input_features)
-    input_features = _convert_woodwork_types_wrapper(input_features.to_dataframe())
+    pipeline_features = pipeline.compute_estimator_features(input_features, y).to_dataframe()
```

Comment:
If I'm understanding this completely, moving this from `_make_single_prediction_shap_table` is where the speedup is coming from?

Reply:
Yes!